### PR TITLE
Document range of storage for Golden Retriever

### PR DESCRIPTION
**Pull request by Murderlon**
Closes https://github.com/transloadit/uppy.io/issues/123

### CODE REVIEW

File: docs/golden-retriever.mdx [hunk 1]
```diff
@@ -32,6 +32,21 @@ If restore is unsuccessful for certain files, they will be marked as “ghosts
...
+
+The amount of storage is dependent upon how much available disk space the device
+has. The general range is between 50MB and 20GB.
+
+| Disk Space | Storage Limit                     |
+| ---------- | --------------------------------- |
+| < 8GB      | 50MB                              |
+| > 8-32GB   | 500MB                             |
+| > 32-128GB | 4% of volume size                 |
+| > 128GB    | 4% or 20GB (whichever is smaller) |
```
Comment (arturi) [inline docs/golden-retriever.mdx:46]: Should we link to MDN or some other source, in case this changes for instance?

Comment (Murderlon) [reply docs/golden-retriever.mdx:46]: There is no single source of truth link, the amount available to service worker caching isn't defined in the specification. The amount varies by browser depending on device and storage conditions. This is a generalised version of the differences and it unlikely to change.

Comment (arturi) [reply docs/golden-retriever.mdx:46]: Found this, wouldn't hurt? https://developer.mozilla.org/en-US/docs/Web/API/Storage_API/Storage_quotas_and_eviction_criteria

Comment (Murderlon) [reply docs/golden-retriever.mdx:46]: Wow I missed that one, that's actually quite good

File: docs/golden-retriever.mdx [hunk 2]
```diff
@@ -32,6 +32,21 @@ If restore is unsuccessful for certain files, they will be marked as “ghosts
...
+
+</details>
```
Comment (Murderlon) [inline docs/golden-retriever.mdx:48]:
```suggestion
Checkout the [storage quotas](https://developer.mozilla.org/en-US/docs/Web/API/Storage_API/Storage_quotas_and_eviction_criteria#other_web_technologies) on MDN to see how much data can be stored depending on the device and browser.
```

**Review by arturi — APPROVED** [review]

**Pull request merged**